hnn_core/check.py

```diff
@@ -0,0 +1,59 @@
+"""Input check functions."""
+
+# Authors: Nick Tolley <nicholas_tolley@brown.edu>
+
+from .params import _long_name
+from .externals.mne import _validate_type, _check_option
+
+
+def _create_gid_list(gids, gid_ranges, valid_cells, arg_name):
```

Comment:
```suggestion
def _check_gids(gids, gid_ranges, valid_cells, arg_name):
```

Just following MNE convention ...